dashboard/lib/projects_list.rb

```diff
@@ -42,5 +84,35 @@ def get_project_row_data(student, project, channel_id)
         updatedAt: project_value['updatedAt'],
       }.with_indifferent_access
     end
+
+    def fetch_published_projects_all_types(limit)
+      PUBLISHED_PROJECT_TYPES.map do |type|
+        [type, fetch_published_projects_by_type(type, limit)]
+      end.to_h
+    end
+
+    def fetch_published_projects_by_type(project_type, limit, offset = 0)
+      PEGASUS_DB[:storage_apps].
+        where(state: 'active', project_type: project_type).
+        exclude(published_at: nil).
+        order(Sequel.desc(:published_at)).
+        limit(limit).
+        offset(offset).
+        map(&method(:get_published_project_data))
```

Comment:
Interesting. I did this because I've been yelled at by rubocop for not using `&:`, but `&:` did not work in this case. I'm happy to keep undiscoverable ruby tricks out of our codebase and do this the way you've shown.